src/com/reco1l/osu/data/Scores.kt

```diff
@@ -228,6 +228,9 @@ interface IScoreInfoDAO {
     @Insert(onConflict = OnConflictStrategy.ABORT)
     fun insertScore(score: ScoreInfo): Long
 
+    @Insert(onConflict = OnConflictStrategy.ABORT)
+    fun insertScores(scores: List<ScoreInfo>): List<Long>
```

Comment:
```suggestion
    fun insertScores(scores: List<ScoreInfo>)
```
We don't need the IDs